File: triatlon-cs/Model/Person/Athlete.cs
using System;

namespace Model.Person
{
    // Athlete

    [Serializable]
    public class Athlete : Person
    {
    
        // Lifecycle

        public Athlete(string firstName, string lastName) : base(firstName, lastName) {}
    
        // Override Methods

        public override string ToString()
        {
            return "Athlete | " + base.ToString();
        }
    
    }
}

File: triatlon-java/model/src/main/java/triatlon/model/person/Athlete.java
package triatlon.model.person;

// Athlete

import java.io.Serializable;

public class Athlete extends Person implements Serializable {

    // Lifecycle

    public Athlete(String firstName, String lastName) {
        super(firstName, lastName);
    }

    // Override Methods

    @Override
    public String toString() {
        return "Athlete | " + super.toString();
    }

}
